daq/faucet_states_collector.py

```diff
@@ -49,12 +49,12 @@ def wrapped(self, *args, **kwargs):
 TOPOLOGY_LINK_MAP = "physical_stack_links"
 TOPOLOGY_LACP = "lacp_lag_status"
 TOPOLOGY_ROOT = "active_root"
-RAW_CONFIG = "raw_config"
+FAUCET_CONFIG = "dps_config"
```

Comment:
but this isn't faucet config either...?  Isn't it just the dps?  Why isn't this just called DPS_CONFIG?

Reply:
PTAL